03week/ticTacToe.js

```diff
@@ -24,23 +24,51 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  //horizontalWin(), checks if there are three X's or O's in [0][0], [0][1], [0][2], etc, indexOf method
+  if (printBoard.indexOf(printBoard) === ((([0][0]) && [0][1] && [0][2]) || ([1][0] && [1][1] && [1][2]) || ([2][0] && [2][1] && [2][2])) {
```

Comment:
Why are you checking for the index of printBoard in printBoard? This would be -1 unless there was an instance of printBoard inside of printBoard...